### DISCUSSION TITLE
Mobiltilpassede komponenter (knapper, checkboks, etc.)

### DISCUSSION THREAD

**Discussion by mariusthedesigner** [fremtind/jokul#2288] [Annet]
Hvilke steg i [prosessen](https://fremtind.github.io/jokul/komigang/prosessen) er utført?
- [X] Undersøke
- [X] Rapportere
- [ ] Avklare
- [ ] Spesifisere krav
- [ ] Utvikle
- [ ] Ferdig!

**Er ønsket tilknyttet et problem? Fortell oss mer!**

Utgangspunktet til en komponent er «Normal» og skal brukes på både desktop og touch-flater (mobil og tablet). Men teksten endrer ikke størrelse basert på skjermstørrelse slik som annen tekst på siden.

**Beskriv løsningen du ønsker**

For å endre på dette må noen ta tak i alle komponentene og tilpasse de til mobi slik at de beholder like stor touch-flate, men får mindre typografi på mobil. Compact-elementer må IKKE brukes på mobil, og må erstattes med «Normal» komponenter.

**Hvilke alternativer har du vurdert?**
Jeg har vurdert en egen mobil-variant, men det ble tatt opp på forum at det var bedre å ta utgangspunkt i Normal-komponenten.

**Mer info**
![Group 1300](https://user-images.githubusercontent.com/59652564/112155520-acee4480-8be5-11eb-9edc-fe56e0e3cfb4.png)

@joms @Murstam

**Comment by lfbergee** [comment]
Dette er relatert til https://github.com/fremtind/jokul/blob/master/packages/text-input/text-input.scss#L69 , som da bør bruke jkl-body-klassen isteden for mixinen, så den får screen-mixinen som setter font-size. Må nok gåes igjennom komponent for komponent for å sjekke at det blir rett :)
